index.js

```diff
@@ -0,0 +1,108 @@
+/*1-Write a console.log statement saying "Hello World!" for each language that you know.*/
+console.log('Halo, dunia!'); //Indoniasia
+console.log('Ciao, mondo!'); //Italian
+console.log('Hola, mundo!'); //Spanish
+
+//2-onsider the following code://
+console.log("I'm awesome");
+//Declare a variable age and initialize it with an integer, using these exact steps://
+var age;
+console.log('The Value of Age will be: undefined');
+console.log(age);
+age= 28;
+console.log('The Value of Age will be: 28');
+console.log(age);
+//4-Declare a variable name and assign a string to it.//
+var name = 'Hamam Alsamel';
+console.log('The Name will be: Hamam Alsamel');
+console.log(name);
+var x= name;
+console.log('The Value of X will be: Hamam Alsamel');
+console.log(x);
+//5-How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+var number= 7.25;
+console.log(number);
+var z= 7;
+console.log(z);
+if (number>=z){
+  console.log(number);
+  var newvalue = number;
+  console.log(newvalue);
+}
+//6- Arrays//
+
+const list = ['','',''];
+console.log('The array has no value becauswe its empty');
+console.log(list);
+
+const favouritanimal = ['Cat','Dog','Birds','elephant'];
+console.log(favouritanimal);
+favouritanimal.push('Kitten')
+console.log(favouritanimal);
+
+//7-More strings: Let's consider the following string: let myString = "this is a test".
+let mystring= "This is a test";
+console.log(mystring);
+console.log(mystring.length);
+console.log('the length of my string');
+
+//8.If x equals 7, and the only other statement is x = x % 3, what would be the new value of x?
+
+var x=7;
+x = x%7;
+console.log(x);
+console.log( x % 7);
+if (x%7===0){
+  console.log("True");
+}
+else {
+  console.log('False');
+}
+
+//9.Write a program to answer the following questions:
+const arrays=['tomate', 5, 'botato'];
+console.log(arrays);
+
+var x=6;
+var y=10;
+if (x/0===y/0) {
+  console.log('Infinities');
+}
+
+//10.I’ve declared a multidimensional array (an array inside an array) -- see below. How can I access the third item’s second element? i.e. [2, 1] is the third element, and I want to access 1. Add a console.log statement accessing this item.
+
+const grid= [[0,1], [1,1], [2,1], [3,1]];
+const item=  grid[2,2];
```

Comment:
We didn't cover this in class, but when working with nested arrays (arrays inside arrays), you need to use two sets of square brackets to access the element like `grid[2][1]`.

For example, if you use `grid[2]`, you get the second element of the grid array `[2, 1]`. So if you use `grid[2][1]`, you get the second element of `grid[2]`, which is 1.
```javascript
grid[2]; // this is [2, 1]
grid[2][1]; // this is 1
```

I see why you'd use a comma like in `[2, 2]`! But that actually prints out `[2, 1]` instead of `1`. I didn't understand why until I realized the comma operator returns the last value, so `grid[2,2]` becomes `[2]` ([explanation](https://stackoverflow.com/questions/7421013/why-does-5-6-8-71-2-8-in-javascript))